### DISCUSSION TITLE
Welcome to Auction_House_V2 Discussions!

### DISCUSSION THREAD

**Discussion by atharmohammad** [atharmohammad/Auction_House_V2#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by YrayPixels** [comment]
What is the progress of the auction house

**Reply by atharmohammad** [reply]
It's completed and live on devnet. There might be some improvements and more features later on but no eta. Also it's not audited yet, at some point if possible would try to get it audited.

**Reply by YrayPixels** [reply]
Okay would love to see this happen, thanks.
